packages/filebrowser/src/listing.ts

```diff
@@ -10,8 +10,6 @@ import {
 
 import { PathExt, Time } from '@jupyterlab/coreutils';
 
-import {} from '@jupyterlab/apputils';
```

Comment:
@echarles did you put it here for a reason, or just a leftover? Introduced in: https://github.com/jupyterlab/jupyterlab/commit/b3cf746ed2ea50a87f7163b3a5e73cf479099a3f

Reply:
@krassowski This was not intended and should be removed. Thx for checking Michal.